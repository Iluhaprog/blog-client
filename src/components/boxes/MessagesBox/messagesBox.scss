.messages-box {
    max-width: 300px;
    max-height: 100vh;
    width: 100%;
    position: absolute;
    top: 10px; 
    right: 30px;
    overflow-y: scroll; 
}