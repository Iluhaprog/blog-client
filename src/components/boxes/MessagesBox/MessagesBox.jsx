import React from 'react';
import './messagesBox.scss';

export default props => (
    <section className='messages-box'>
        { props.children }
    </section>
)