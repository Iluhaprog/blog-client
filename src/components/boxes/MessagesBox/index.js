import MessagesBox from './MessagesBox';

export { 
    MessagesBox
};