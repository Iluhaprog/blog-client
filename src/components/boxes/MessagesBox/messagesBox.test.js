import React from 'react';
import { render } from '@testing-library/react';
import MessagesBox from './MessagesBox';

test('Test view of messages box', () => {
    const { asFragment } = render(<MessagesBox>child</MessagesBox>);
    expect(asFragment()).toMatchSnapshot();
});