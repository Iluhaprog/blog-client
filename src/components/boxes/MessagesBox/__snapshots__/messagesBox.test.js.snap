// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test view of messages box 1`] = `
<DocumentFragment>
  <section
    class="messages-box"
  >
    child
  </section>
</DocumentFragment>
`;
